utils/gha-runners/createNamespaceConfig.sh

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env bash
```

Comment:
Please do not run commands under sudo with gha.
Please customize and use the namespace creation scripts from https://github.com/pmem/dev-utils-kit.
In this repository you only need to run the set testconfigs on the corresponding runners.